### PR TITLE
New Week 2

### PR DESCRIPTION
**Pull request by Rajeev2k11**
Assignment -1  from week-2 completed

### CODE REVIEW

File: Week-2/Assignment-1/ques-3.html
Comment (NishantNaithani) [inline Week-2/Assignment-1/ques-3.html:1]: Here,
Default value is not defined

**Review by NishantNaithani — CHANGES_REQUESTED** [review]

**Comment by NishantNaithani** [comment]
Hey @Rajeev2k11 ,
Added comments

File: Week-2/Assignment-1/ques-3.html
Comment (Rajeev2k11) [reply Week-2/Assignment-1/ques-3.html:1]: Which default value? I could not get it.

Comment (NishantNaithani) [reply Week-2/Assignment-1/ques-3.html:1]: Screenshot 2023-09-01 at 6 29 55 PM

**Pull request merged**